### DISCUSSION TITLE
What is Collove?!

### DISCUSSION THREAD

**Discussion by Strahfe** [collove/pasteme#50] [Q&A] [answered]
I would be wondering if you describe more about Collove organization.

**Comment by lnxpy** [comment, marked as answer]
Hey Arshia. How have you been?! Collove is an open-source community where people collaborate to build cool tools. Col + Love (Co-love) is the combination of two words; Collaboration and Love which means, a place where everyone pushes lovely commits only. :))

Let me know if you want to be part of this community. I would be so glad to have you here.

**Reply by Strahfe** [reply]
Cool. Thank you.
